libbeat/otelbeat/oteltranslate/outputs/elasticsearch/config_otel.go

```diff
@@ -123,19 +125,22 @@ func ToOTelConfig(output *config.C) (map[string]any, error) {
 		return nil, fmt.Errorf("cannot convert SSL config into OTel: %w", err)
 	}
 
-	otelYAMLCfg := map[string]any{
-		"logs_index":  escfg.Index,        // index
-		"endpoints":   hosts,              // hosts, protocol, path, port
-		"num_workers": escfg.NumWorkers(), // worker/workers
+	// get number of workers
+	var worker int // Default value is 1
+	if escfg.NumWorkers() < 1 {
+		worker = 1
+	} else {
+		worker = escfg.NumWorkers()
+	}
```

Comment:
Could do it like this:

```
	workers := 1
	if escfg.NumWorkers() > 1 {
		worker = escfg.NumWorkers()
	}
```

Also workers sounds better than `worker` in this context.